backend/internal/environment/environment.go

```diff
@@ -95,6 +97,9 @@ func init() {
 	setOrFallback(EnvVarDatabaseHost, DefaultHostDatabase)
 	setOrFallback(EnvVarDatabaseUser, "")
 	setOrFallback(EnvVarDatabasePassword, "")
+
+	// Create a global logger
+	zap.ReplaceGlobals(zap.Must(zap.NewProduction()))
```

Comment:
I would place this at the beginning of the `main` function since this file deals only with environment variables.